From 590bd5f81207bb55bd02b4a37f73a1f64dac452d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 23 Jan 2023 13:20:58 -0800
Subject: [PATCH] parser.h: Including missing <cstdint>

This is needed from gcc-13+ see [1]

Fixes

../git/framework/platform/parser.h:282:43: error: 'uint32_t' does not name a type
  282 |                 *type       = static_cast<uint32_t>(number);
      |                                           ^~~~~~~~
../git/framework/platform/parser.h:21:1: note: 'uint32_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?

[1] https://www.gnu.org/software/gcc/gcc-13/porting_to.html

Upstream-Status: Submitted [https://github.com/KhronosGroup/Vulkan-Samples/pull/602]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 framework/platform/parser.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/framework/platform/parser.h b/framework/platform/parser.h
index 215f3d80..a5bd1344 100644
--- a/framework/platform/parser.h
+++ b/framework/platform/parser.h
@@ -18,6 +18,7 @@
 #pragma once
 
 #include <cassert>
+#include <cstdint>
 #include <string>
 #include <typeindex>
 #include <vector>
-- 
2.39.1

